tests/drivers/gpio/gpio_basic_api/testcase.yaml

```diff
@@ -20,3 +20,11 @@ tests:
   drivers.gpio.mr_canhubk3_wkpu:
     platform_allow: mr_canhubk3
     extra_args: "DTC_OVERLAY_FILE=boards/mr_canhubk3_wkpu.overlay"
+
+  drivers.gpio.2pin_ke17z_fgpio:
+    min_flash: 34
+    filter: dt_compat_enabled("test-gpio-basic-api")
+    platform_allow:
+      - frdm_ke17z
+      - frdm_ke17z512
+    extra_args: "DTC_OVERLAY_FILE=boards/frdm_ke17z_fgpio.overlay"
```

Comment:
please add fgpio_loopback
`fixture:
   - fgpio_loopback`

Reply:
also please move the gpio test to d0 and d1 which is j1 2-4. and avoid board conflicts with i2c. Thanks